src/dotnet/HQ.Abstractions/Times/DeleteTime.cs

```diff
@@ -0,0 +1,13 @@
+namespace HQ.Abstractions.Times {
+
+public class DeleteTimeV1
+{
+   public class Request
+    {
+        public Guid? Id { get; set; }
+        public Guid? StaffId { get; set; }
```

Comment:
Because we do the authorization checks in the controller, we don't need the StaffId on the request. We just need the time ID.